.travis.yml

```diff
@@ -4,7 +4,7 @@ services:
 - docker
 language: go
 go:
-  - "1.13.x"
+  - "1.18.x"
```

Comment:
I don't think Travis is enabled on this repo.  We should switch to GitHub Actions, but that doesn't need to happen in this PR.

Reply:
Yes, we need that, so it does not leave us with issues such as recent fmt error on `tools.go`.